protected/humhub/modules/comment/Module.php

```diff
@@ -97,4 +97,29 @@ public function canComment($object)
         return true;
     }
 
+    /**
+     * Checks if given content object is a subcomment
+     *
+     * @param $object
+     * @return bool
+     */
+    public function isSubComment($object)
+    {
+        if ($object instanceof Comment && $object->object_model === Comment::class) {
+            return true;
+        }
+        return false;
+    }
+
+    /**
+     * Return parent of given content object
+     *
+     * @param $object
+     * @return ContentActiveRecord
```

Comment:
We can remove this function you can just call `$object->getPolymorphicRelation()`. Also I'd suggest always to use `findOne(['id' => $id])` since its more safe, probably not a problem in this case but we should stick to it.

Reply:
Done! Thank you for your suggestion.